account_payment_cc/__openerp__.py

```diff
@@ -0,0 +1,88 @@
+﻿# -*- coding: utf-8 -*-
+
+{
+    "name": "Credit Card Payments",
+    "version": "1.7",
+    "category": "Accounting",
+    "author": 'Ursa & OpenERP SA',
+    "summary": "Adds support for Credit Card AP movements. (Ursa)",
+    "description": """
```

Comment:
The description should match the template in https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst